### DISCUSSION TITLE
Can't validate Docker Volume

### DISCUSSION THREAD

**Discussion by kfowlks** [mealie-recipes/mealie#1469] [Q&A]
I'm running an almost stock docker-compse config just with some changes to the volumes path because I'm running this container on UNRAID. The site comes up but when I attempt to validate the volumes it fails. I did notice that the log appears to be looking for the database in the "/app/data" directory. How can I override this using the values below in my config. 

Thanks

Excerpt from Log file:

 "DB_ENGINE": "sqlite",        
              "DB_PROVIDER": {
                  "data_dir": "/app/data",
                 "prefix": ""
     },

Docker Compose Config
---
version: "3.7"
services:
  mealie-frontend:
    image: hkotel/mealie:frontend-v1.0.0beta-3
    container_name: mealie-frontend
    environment:
    # Set Frontend ENV Variables Here
      - API_URL=http://mealie-api:9000 # 
    restart: always
    ports:
      - "9925:3000" # 
    volumes:
      - mealie-data:/mnt/user/appdata/mealie/ #
      #- mealie-data:/app/data/ # 
  mealie-api:
    image: hkotel/mealie:api-v1.0.0beta-3
    container_name: mealie-api
    volumes:
      - mealie-data:/mnt/user/appdata/mealie/
      #- mealie-data:/app/data/
      
    environment:
    # Set Backend ENV Variables Here
      - ALLOW_SIGNUP=true
      - PUID=1000
      - PGID=1000
      - TZ=America/Detroit
      - MAX_WORKERS=1
      - WEB_CONCURRENCY=1
      - BASE_URL=https://mealie.yourdomain.com
    restart: always

volumes:
  mealie-data:
    driver: local

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened; if you're still having issues, feel free to open an issue for it
